packages/nodes-base/nodes/Calendly/GenericFunctions.ts

```diff
@@ -57,37 +65,10 @@ export async function calendlyApiRequest(
 		delete options.qs;
 	}
 	options = Object.assign({}, options, option);
-	return await this.helpers.requestWithAuthentication.call(this, 'calendlyApi', options);
-}
-
-export async function validateCredentials(
```

Comment:
I removed this function because I saw it was not being called

Reply:
This is a good find this will be from the old credential test process and we just didn't clean it up, Thanks for leaving the code in a better state than you found it